dartagnan/src/main/java/com/dat3m/dartagnan/program/analysis/alias/AliasAnalysis.java

```diff
@@ -38,9 +43,32 @@ final class Config {
                 description = "General type of analysis that approximates the 'loc' relationship between memory events.")
         private Alias method = Alias.FIELD_SENSITIVE;
 
+        private final boolean addEqualityAnalysis = true;
```

Comment:
Why not adding this as a configuration option?

Reply:
I didn't add it (yet) because I was not sure if this option is so obvious that it should simply be always enabled.
I would rather avoid cluttering the configuration settings with options that are no-brainers.
On the other hand, the read-relaxation should become configurable.

Reply:
I agree about not cluttering the configurations ... but if it will always be enabled, we don't need this filed at all

Reply:
Yes, I can remove it completely. For debugging/testing purposes, it was very useful though.